frontend/asset_mapper.rst

```diff
@@ -626,15 +626,14 @@ To make your AssetMapper-powered site fly, there are a few things you need to
 do. If you want to take a shortcut, you can use a service like `Cloudflare`_,
 which will automatically do most of these things for you:
 
-- **Use HTTP/2**: Your web server should be running HTTP/2 (or HTTP/3) so the
+- **Use HTTP/2**: Your web server should be running HTTP/2 or HTTP/3 so the
   browser can download assets in parallel. HTTP/2 is automatically enabled in Caddy
   and can be activated in Nginx and Apache. Or, proxy your site through a
   service like Cloudflare, which will automatically enable HTTP/2 for you.
 
 - **Compress your assets**: Your web server should compress (e.g. using gzip)
   your assets (JavaScript, CSS, images) before sending them to the browser. This
   is automatically enabled in Caddy and can be activated in Nginx and Apache.
-  In Cloudflare, assets are compressed by default.
```

Comment:
Why did you remove this?

Cloudflare still compress by default (in gzip, brotli or zstandard) all web assets and more. See https://developers.cloudflare.com/speed/optimization/content/compression/

What Cloudflare remove was the minification of assets. See https://community.cloudflare.com/t/deprecating-auto-minify/655677 But we already removed that from the Symfony Docs.

Reply:
While merging, I reverted this change.